.env.example

```
@@ -10,15 +10,15 @@ OUT_DIR=/app/out
 NUM_WORKERS=1
 SPARK_MASTER=local[1]
 SPARK_LOCAL_DIR=/tmp/spark
-PUBLIC_API_URL=http://${SERVER_C_HOST}:8282
+PUBLIC_API_URL=http://{SERVER_C_HOST}:8282
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PUBLIC_API_URL key should go before the SPARK_LOCAL_DIR key

(UnorderedKey)